src/app/components/home/profile/edit-profile/edit-profile.component.ts

```diff
@@ -23,17 +23,15 @@ export class EditProfileComponent implements OnInit {
 
   @Input() currentUser: BasicUser;
 
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
what rules does this disable ? or better yet what does ban-types do ? this seems fishy